pkgs/development/compilers/ghc/common-hadrian.nix

```
@@ -271,8 +259,20 @@
         })
       ]
 
-      # Missing ELF symbols
-      ++ lib.optionals stdenv.targetPlatform.isAndroid [
+      # Unconditionally applied while we wait for upstream developments
+      # (issue/fix/merge/release) to inform specific guards
+      ++ [
+        # https://gitlab.haskell.org/ghc/ghc/-/merge_requests/13919
+        (fetchpatch {
+          name = "include-modern-utimbuf.patch";
+          url = "https://gitlab.haskell.org/ghc/ghc/-/commit/7e75928ed0f1c4654de6ddd13d0b00bf4b5c6411.patch";
+          hash = "sha256-sb+AHdkGkCu8MW0xoQIpD5kEc0zYX8udAMDoC+TWc0Q=";
+        })
+
+        # https://gitlab.haskell.org/ghc/ghc/-/issues/26290
```

I believe this might need:


```suggestion
        # https://gitlab.haskell.org/ghc/ghc/-/issues/26290 krank:ignore-line
```

I have not run krank myself, though.

---

Yes.